.ibm/pipelines/windows-test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash 
+set -x
+apt-get update -y
+apt-get install -y sshpass
```

Comment:
It would be nice to install in the Docker image used for the pipeline, from this Dockerfile: https://github.com/redhat-developer/odo/blob/main/.ibm/images/Dockerfile

Reply:
Understood, I will add it in Docker file. AFAIU, currently we are using the image hosted in your registry, right?

Reply:
Yes, do we have annother registry shared by the team that we can use easily?

Reply:
I think we can use the odo quay.io account to store this image.
@kadel WDYT?